tomviz/OperatorPython.cxx

```diff
@@ -377,4 +426,15 @@ EditOperatorWidget* OperatorPython::getEditorContents(QWidget* p)
 {
   return new EditPythonOperatorWidget(p, this);
 }
+
+void OperatorPython::createNewChildDataSource(
+  const QString& label, vtkSmartPointer<vtkSMProxy> producerProxy)
+{
+
+  DataSource* childDS = new DataSource(
```

Comment:
I think the VTK object leaks may be coming from here. The child `DataSource` parent should be `this`, the operator instance. Doing so will make the child data source get deleted with the `OperatorPython` is deleted.